src/shared/async.c

```diff
@@ -41,12 +49,23 @@ int asynchronous_sync(pid_t *ret_pid) {
 static int close_func(void *p) {
         unsigned v = PTR_TO_UINT(p);
 
-        (void) prctl(PR_SET_NAME, (unsigned long*) "(sd-close)");
-
         /* Note: 💣 This function is invoked in a child process created via glibc's clone() wrapper. In such
          *       children memory allocation is not allowed, since glibc does not release malloc mutexes in
          *       clone() 💣 */
 
+        /* This is a port man's version of rename_process(). We don't use the real thing to avoid any memory
```

Comment:
"a port man" as in a sailor, or as in a dock worker? I feel the comment should disambiguate

Reply:
could also be a "port wine maker" I guess - this really needs some clarifications

Reply:
maybe a port wine drinker?